app/services/liquid_assigns_service.rb

```diff
@@ -15,4 +15,27 @@ def assigns
 
   def assigns_pro
   end
+
+  def project_drops
+    {
+      'evidence' => project_records(type: :evidence),
+      'issues' => project_records(type: :issue),
+      'nodes' => project_records(type: :node),
+      'notes' => project_records(type: :note),
+      'project' => ProjectDrop.new(project),
+      'tags' => project_records(type: :tag)
+    }
+  end
+
+  def project_records(type:)
```

Comment:
What about naming this one `project_record_drops` since it creates the drops

Reply:
I went with `cached_drops` instead since "caching" seems to be the main job of the method